README.md

```diff
@@ -100,20 +100,6 @@ In order to be useful, Anon needs to be told what you want to do to each column
 }
 ```
 
-## How to contribute
```

Comment:
A lot of repos leave a contributing summary in the main readme e.g. [Scala](https://github.com/scala/scala), [Akka](https://github.com/scala/scala#how-to-contribute) and [Chronos](https://github.com/mesos/chronos#contributing). Could have a 2-3 line summary or just a pointer to the contributing file itself, YMMV.

Reply:
good point